sklearn/preprocessing/tests/test_data.py

```diff
@@ -2506,3 +2506,63 @@ def test_standard_scaler_sparse_partial_fit_finite_variance(X_2):
     scaler = StandardScaler(with_mean=False)
     scaler.fit(X_1).partial_fit(X_2)
     assert np.isfinite(scaler.var_[0])
+
+
+def test_minmax_scaler_cupy():
+    cp = pytest.importorskip("cupy")
+    X_np = iris.data
+    X_cp = cp.asarray(X_np)
+
+    scaler = MinMaxScaler(copy=True)
+    t_X_cp = scaler.fit_transform(X_cp)
+    assert type(t_X_cp) == type(X_cp)
```

Comment:
maybe also check that the min and max values of `t_X_cp` are 0 and 1.

Reply:
Done